py/private/py_binary.bzl

```diff
@@ -149,6 +149,10 @@ A collision can occour when multiple packages providing the same file are instal
         default = "error",
         values = ["error", "warning", "ignore"],
     ),
+    "interpreter_args": attr.string_list(
```

Comment:
nit: from https://docs.python.org/3/using/cmdline.html#command-line it seems like it should be named `interpreter_options` rather than args

Reply:
> nit: from https://docs.python.org/3/using/cmdline.html#command-line it seems like it should be named `interpreter_options` rather than args

agree, updated